models.html

```diff
@@ -460,6 +469,142 @@ <h2>Constituency Parsing</h2>
+                  <div class="tab__nav__item__glyph">
+                    <svg>
+                      <use xlink:href="#icon__disclosure"></use>
+                    </svg>
+                  </div>
+                </li>
+                <li data-tab="evaluation" class="tab__nav__item"><span>Evaluation</span>
+                  <div class="tab__nav__item__glyph">
+                    <svg>
+                      <use xlink:href="#icon__disclosure"></use>
+                    </svg>
+                  </div>
+                </li>
+                <li data-tab="training" class="tab__nav__item"><span>Training</span>
+                  <div class="tab__nav__item__glyph">
+                    <svg>
+                      <use xlink:href="#icon__disclosure"></use>
+                    </svg>
+                  </div>
+                </li>
+                <li class="tab__nav__link">
+                  <a href="http://demo.allennlp.org/dependency-parsing" target="_blank">
+                    <span>View Demo</span>
+                  </a>
+                </li>
+
+              </ul>
+              <div class="tab__page-container">
+                <div data-tab="prediction" class="tab__page">
+                  <div class="tab__page__content"><pre><code class="no-highlight">echo '{"sentence": "If I bring 10 dollars tomorrow, can you buy me lunch?"}' > dparse-examples.jsonl
+allennlp predict \
+    https://s3-us-west-2.amazonaws.com/allennlp/models/biaffine-dependency-parser-ptb-2018.08.23.tar.gz \
+    dparse-examples.jsonl</code></pre></div>
+                </div>
+                <div data-tab="evaluation" class="tab__page">
+                  <div class="tab__page__content">
+                    <p class="t-sm">The dependency parser was evaluated on the Penn Tree Bank dataset. Unfortunately we cannot release this data due to licensing restrictions by the LDC. You can download the PTB data <a href="https://catalog.ldc.upenn.edu/ldc99t42" target="_blank">from the LDC website</a>.</p>
+                  </div>
+                </div>
+                <div data-tab="training" class="tab__page">
+                  <div class="tab__page__content">
+                    <p class="t-sm">The dependency parser was evaluated on the Penn Tree Bank dataset. Unfortunately we cannot release this data due to licensing restrictions by the LDC. You can download the PTB data <a href="https://catalog.ldc.upenn.edu/ldc99t42" target="_blank">from the LDC website</a>.</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+
+      <!-- Open Information Extraction-->
+      <div class="banner banner--hr">
+        <div class="constrained constrained--med">
+          <div class="padded-med">
+            <h2 id="open-information-extraction">Open Information Extraction</h2>
+            <p class="t-sm">
```

Comment:
Can you add some performance statistics here?

Reply:
Done.